tests/e2e/specs/modules/analytics/write-scope-requests.test.js

```diff
@@ -238,6 +239,9 @@ describe( 'Analytics write scope requests', () => {
 			text: /set up a new property/i,
 		} );
 
+		// Add a brief delay to allow the submit button to become enabled.
+		await pageWait();
```

Comment:
This change was mostly unrelated to the other changes here as far as I can tell but the failure caused confusion here so it seems to be fixed now 👍